config/routes.rb

```diff
@@ -36,4 +37,15 @@
   get "/veterinary_offices/:veterinary_office_id/veterinarians", to: "veterinary_offices#veterinarians"
   get "/veterinary_offices/:veterinary_office_id/veterinarians/new", to: "veterinarians#new"
   post "/veterinary_offices/:veterinary_office_id/veterinarians", to: "veterinarians#create"
+
+  get "/applications/new", to: "applications#new"
+  get "/applications/:id", to: "applications#show"
+  post "/applications", to: "applications#create"
+  patch "/applications/:id", to: "applications#update"
+
+  post "/application_pet", to: "application_pet#create"
```

Comment:
This is extremely minor but in a joins controller, it's conventional to pluralize the second record, i.e. `application_pets` (while application stays singular)